core-kotlin-modules/core-kotlin-strings-3/src/main/kotlin/com/baeldung/string/extension_functions/StringExtensionFunctionClass.kt

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.string.extension_functions
+
+import java.nio.charset.Charset
+
+class StringExtensionFunctionClass {
```

Comment:
Hmm, it feels weird to wrap the examples in this 'helper' class - why not just use the extension functions directly in the test?